15_simulations/index.tex

```diff
@@ -1 +1,93 @@
+
+
+\begin{itemize}
+    \item \textbf{Archipelago Survivability}: Turns until all the islands in the archipelago die.
+    \item \textbf{First Island Death}: Turns until any island dies.
+    \item \textbf{Island Resources$^1$$^2$}: Resource an island has each turn.
+    \item \textbf{Gini Index}: A measure of how fair the distribution of resources is across the archipelago.
+    \item \textbf{Disasters Survived}: Number of disasters the archipelago has survived.
+    \item \textbf{Island Trading$^1$$^2$}: Resources an island has gifted to other islands.
+    \item \textbf{Average Disaster Damage Mitigated(ADDM)}: Average disaster damage mitigated by the common pool.
+    \item \textbf{Island Foraging Statistics(IFS)$^1$}: Resources an island has invested and gained from foraging.
+    \item \textbf{Archipelago Foraging Sustainability(AFS)}: The average net forage returns across all islands.
+    \item \textbf{IIGO Roles$^2$}: The power each island has at any turn in the game. Power in this case is occupying one of the three roles in IIGO.
+    \item \textbf{IIGO Allocations$^1$$^2$}: Amount of resources allocated to each island by the president and the amount the island has taken.
+    \item \textbf{IIGO Tax$^1$$^2$}: Amount of resources an island is expected to pay to the common pool for tax and amount the island has paid.
+    \item \textbf{IIGO Sanctions$^1$$^2$}: Amount of resources an island is expected to pay to the common pool for sanctions and amount the island has paid.
+    
+\end{itemize}
+\small{$^1$ Presented on a per island basis. $^2$ Presented as a graph}
+
+\section{Baseline Simulation}
+\label{sec:Simulations:baseline}
+
+In order to make any meaningful analysis one must first have a baseline from which to compare. In our case, this baseline is a \emph{normal} set of environment conditions which allows the agents to thrive for a significant period of time, chosen at around 100 turns. From this baseline we can then start adjusting the configuration of the simulation, either making disasters more frequent/deadly, or making resources more sparse.
+
+Note: If a metric is not of interest to a certain area of exploration it will not be shown.
+
+\subsection{Baseline Numeric Metrics}
+\label{subsec:Simulations:baseline:num_metrics}
+\begin{table}[htb]
+    \centering
+    \begin{tabular}{|l|l|}
+    \hline
+    \textbf{Metric}                     & \textbf{Value} \\ \hline
+    \textbf{Archipelago Survivability}  &       \\
+    \textbf{First Island Death}         &       \\
+    \textbf{Gini Index}                 &       \\
+    \textbf{Disasters Survived}         &       \\
+    \textbf{ADDM}                       &       \\
+    \textbf{AFS}                        &       \\ \hline
+\end{tabular}
+\caption{Numerical Metrics}
+\end{table}
+
+\subsection{Baseline IFS}
+\label{subsec:Simulations:baseline:ifs}
+\begin{table}[htb]
```

Comment:
hm.. the widths of the columns are not ideal.
can you enlarge the width of the very first column xxx